src/tasks/minions/fishingActivity.ts

```diff
@@ -11,11 +11,26 @@ import { anglerBoostPercent, rand, roll } from '../../lib/util';
 import { handleTripFinish } from '../../lib/util/handleTripFinish';
 import itemID from '../../lib/util/itemID';
 
+function radasBlessing(user: KlasaUser) {
+	const skillingGear = user.getGear('skilling');
+	if (skillingGear.hasEquipped(itemID("Rada's Blessing 4"))) {
+		return { blessingEquipped: true, blessingChance: 8 };
+	} else if (skillingGear.hasEquipped(itemID("Rada's Blessing 3"))) {
+		return { blessingEquipped: true, blessingChance: 6 };
+	} else if (skillingGear.hasEquipped(itemID("Rada's Blessing 2"))) {
+		return { blessingEquipped: true, blessingChance: 4 };
+	} else if (skillingGear.hasEquipped(itemID("Rada's Blessing 1"))) {
+		return { blessingEquipped: true, blessingChance: 2 };
+	}
+	return { blessingEquipped: false, blessingChance: 0 };
+}
```

Comment:
do something like
```ts
const blessingBoosts = [["Rada's blessing 4", 8]];

for (const [itemName, boostPercent] of blessingBoosts) {
  if (hasitemequippedanywhere(itemname)) {
    return {whatever}
  }
}
```
instead of chaining if/else like this

Reply:
Updated thanks :)